src/woo_search/conf/base.py

```diff
@@ -67,6 +73,34 @@
 # Default (connection timeout, read timeout) for the requests library (in seconds)
 REQUESTS_DEFAULT_TIMEOUT = (10, 30)
 
+#
+# Elasticsearch DSL custom settings
+#
+ELASTICSEARCH_DSL_HOSTS = config(
+    "ELASTICSEARCH_DSL_HOSTS", default=["http://localhost:9200/"]
+)
+ELASTICSEARCH_DSL_TIMEOUT = config("ELASTICSEARCH_DSL_TIMEOUT", default=60)
+
+ELASTICSEARCH_USER = config("ELASTICSEARCH_USER", default="elastic")
+ELASTICSEARCH_SECRET = config("ELASTICSEARCH_SECRET", default="insecure-elastic")
```

Comment:
```suggestion
ELASTICSEARCH_USER = config("ELASTICSEARCH_USER", default="")
ELASTICSEARCH_SECRET = config("ELASTICSEARCH_SECRET", default="")
```
better to define the envvar in `docker-compose` instead of setting defaults in the settings so that if nothing is specified, you get the appropriate errors (if auth is required).